packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -212,6 +213,28 @@ export async function importBlock(
   const newHead = this.recomputeForkChoiceHead(ForkchoiceCaller.importBlock);
   const currFinalizedEpoch = this.forkChoice.getFinalizedCheckpoint().epoch;
 
+  // Suppress fcu call if this is true
+  let shouldOverrideFcu = false;
+
+  if (opts.isGossipBlock && isExecutionStateType(postState)) {
+    const proposerIndex = postState.epochCtx.getBeaconProposer(blockSlot + 1);
+    const feeRecipient = this.beaconProposerCache.get(proposerIndex);
+
+    if (feeRecipient) {
+      // We would set this to true if
+      //  1) This is a gossip block
+      //  2) We are proposer of next slot
+      //  3) Proposer boost reorg related flag is turned on (this is checked inside the function)
+      //  4) Block meets the criteria of being re-orged out (this is also checked inside the function)
+      shouldOverrideFcu = this.shouldOverrideForkChoiceUpdate(this.clock.currentSlot, blockSummary.blockRoot);
+    }
+  }
+
+  if (shouldOverrideFcu) {
+    this.logger.info("Weak block detected. Skip fcu call in importBlock");
```

Comment:
should include block root(s) in log, maybe even the slot so it will show up if you filter by slot

Reply:
this is more like technical information, maybe change to `verbose` level? `info` log is for users

Reply:
also we wanna add a `gauge` metric in this case, that'll help us investigate later